src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -288,9 +296,11 @@ export const FlowChartWrapper = ({
           >
             <LoadingIcon visible={loading} />
           </div>
+          <ShareableUrlMetadata />
         </div>
         <ExportModal />
         <MetadataModal />
+        <ShareableUrlModal />
```

Comment:
could we do  
{ isRunningLocal() && <ShareableUrlModal /> here then we don't need to in the component?

Reply:
Updated.